docker/dockerfiles/python-rolelive

```diff
@@ -1,7 +1,5 @@
-FROM python:3.7-alpine
+FROM python:latest
```

Comment:
We still should freeze the Python version, I just asked to use a more recent one. Use 3.9